components/dataList.js

```diff
@@ -67,7 +67,7 @@ export default [
           desc: ""
         },
         {
-          name: "Oatmeal",
+          name: "Oatmeall",
```

Comment:
It should be Oatmeal

Reply:
Ops, must be a typo while editing the file, my apologies.

Reply:
it's okay. commit the after resolving it.